### PR TITLE
Update README with more explicit instructions (venv)

### PR DESCRIPTION
**Pull request by joeleba**
**What this PR does and why we need it:**

Update the README with more explicit instructions, especially regarding virtual environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,19 @@
...
 
-Before you do anything, install the dependencies:
-
 ```
-# pip or pip3, depends on your setup.
+# Clone bazel-bench.
+$ git clone https://github.com/bazelbuild/bazel-bench.git
+$ cd bazel-bench
+
+# Create and activate the virtual environment.
```
Comment (meisterT) [inline README.md:21]: I never did this? What value does it add?

Comment (joeleba) [reply README.md:21]: It keeps the bazel-bench deps contained within this virtual environment and doesn't pollute your global env.

Comment (meisterT) [reply README.md:21]: Let's add then that it's an optional, but recommended step?

Comment (joeleba) [reply README.md]: Done.

**Review by meisterT — APPROVED** [review]

**Pull request merged**